content/applications/finance/expenses/reimburse.rst

```diff
@@ -0,0 +1,134 @@
+:guilabel:`Register Payment` pop-up window appears. Enter the following information in the pop-up
+window:
+
+- :guilabel:`Journal`: Select the accounting journal to post the payment to using the drop-down
+  menu.
+- :guilabel:`Payment Method`: Select how the payment is made using the drop-down menu. The default
+  options are :guilabel:`Manual`, :guilabel:`Checks`, and :guilabel:`NACHA`.
+- :guilabel:`Recipient Bank Account`: Select the employee's bank account that the payment is being
+  sent to. If the employee has a bank account on file in the :ref:`Private Information tab
+  <employees/private-info>` of the **Employees** app, that bank account populates this field, by
+  default.
+- :guilabel:`Amount`: The total amount being reimbursed populates this field, by default. The
+  currency, located to the right of the amount field, can be modified using the drop-down menu.
+- :guilabel:`Payment Date`: Enter the date the payments are issued. The current date populates this
+  field by default.
+- :guilabel:`Memo`: The text entered in the :doc:`Expense Report Summary
+  <../expenses/expense_reports>` field of the expense report populates this field, by default.
+
+.. image:: reimburse/payment.png
+   :align: center
+   :alt: The Register Payment pop-up window filled out for an individual expense report
+         reimbursement.
+
+When the form is completed, click the :guilabel:`Create Payment` button to register the payment, and
+reimburse the employee.
+
+.. _expenses/reimburse-bulk:
+
+Reimburse in bulk
+=================
+
+To reimburse multiple expense reports at once, navigate to :menuselection:`Expenses app --> Expense
+Reports --> Reports To Pay` to view all expense reports in a list view. Next, adjust the
+:guilabel:`STATUS` filters on the far-left side to only present expense reports with a status of
+:guilabel:`POSTED`.
+
+.. tip::
+   Adjusting the :guilabel:`STATUS` filters to only show :guilabel:`POSTED` expense reports is not
+   necessary, but removes the step of selecting each individual report in the list.
+
+Tick the checkbox next to :guilabel:`Employee` to select all the reports in the list. Once ticked,
+the number of selected expense reports appears on a :guilabel:`(#) Selected` button in the
+top-center of the screen. Additionally, a :guilabel:`Register Payment` smart button also appears, on
+the left-hand side.
+
+.. image:: reimburse/multiple-reports.png
+   :align: center
+   :alt: Expense reports filtered by the status Posted, making the Register Payment button visible.
+
+Click the :guilabel:`Register Payment` button, and a :guilabel:`Register Payment` pop-up window
+appears. Enter the following information in the pop-up window:
+
+- :guilabel:`Journal`: Select the accounting journal to post the payment to, using the drop-down
+  menu.
+- :guilabel:`Payment Method`: Select how the payment is made using the drop-down menu. The default
+  options are :guilabel:`Manual`, :guilabel:`Checks`, and :guilabel:`NACHA`.
```

Comment:
Again, I'm also seeing SEPA Credit Transfer as an option here -- but, perhaps, that's only for BE-based businesses. Still, something to look into during your revisions.